### PR TITLE
Process reminders

### PR DESCRIPTION
**Pull request by MilenaHristova**

### CODE REVIEW

File: src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs [hunk 1]
```diff
@@ -54,15 +60,20 @@ public abstract class PullRequestActor : IPullRequestActor
         _logger = loggerFactory.CreateLogger(GetType());
         _redisCache = redis.GetDatabase();
         _actorId = actorId;
+        _reminders = reminders;
 
         PullRequestRedisKey = PullRequest + _actorId.Id;
         PullRequestUpdateRedisKey = PullRequest + _actorId.Id;
```
Comment (premun) [inline src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs]:
```suggestion
        PullRequestUpdateRedisKey = PullRequestUpdate + _actorId.Id;
```

File: src/Maestro/Maestro.ContainerApp/Actors/IReminderManager.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace Maestro.ContainerApp.Actors;
 
 public interface IReminderManager
```
Comment (premun) [inline src/Maestro/Maestro.ContainerApp/Actors/IReminderManager.cs:11]: nit: I think this should be called `ReminderManager.cs`

File: src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs [hunk 3]
```diff
@@ -54,15 +60,20 @@ public abstract class PullRequestActor : IPullRequestActor
...
         PullRequestCheckRedisKey = PullRequestCheck + _actorId.Id;
+        PullRequestCheckReminder = PullRequestCheckReminderPrefix + _actorId.Id;
+        PullRequestUpdateReminder = PullRequestUpdateReminderPrefix + _actorId.Id;
     }
 
     protected string PullRequestRedisKey { get; }
     protected string PullRequestUpdateRedisKey { get; }
     protected string PullRequestCheckRedisKey { get; }
+    protected string PullRequestCheckReminder { get; }
+    protected string PullRequestUpdateReminder { get; }
```
Comment (premun) [inline src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs:76]: Why not just 
```csharp
protected string PullRequestCheckReminder => PullRequestCheckReminderPrefix + _actorId.Id;
```

Comment (MilenaHristova) [reply src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs:76]: I made it the same as PullRequestUpdateRedisKey and the others

Comment (premun) [reply src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs:76]: Yeah, those are also not great. But what the hell, it's a hackathon :)

**Pull request merged**